source/user-manual/reference/ossec-conf/office365-module.rst

```diff
@@ -0,0 +1,256 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+.. _office365-module:
+
+office365
+=========
+
+.. versionadded:: 4.3.0
+
+.. note::
+
+    This module only works on Windows, Linux and MacOS. It is recommended to have it enabled only in one agent to avoid repeated logs.
```

Comment:
```suggestion
    This module only works on Windows, Linux, and macOS. It is recommended to have it enabled only in one agent to avoid repeated logs.
```